handlerGithub/github.go

```diff
@@ -114,15 +126,20 @@ func convertToGithubRepositorySchema(repositoryPayload github.RepositoryPayload)
 			State:        "Created", // This one should be bound to some kind of function return call like that it was successfully created ? Like return error in any of these function, if not then we return default like create
 			ReconsiledAt: time.Now().UTC().String(),
 		},
-		Components: Components{},
 	}
-	// there should be an if statement here reading if the component is enabled
-	//	sonarQubeComponent, err := sonarqube.OnboardSonarQube(repositoryPayload)
-	//	if err != nil {
-	//		return nil, err
-	//	}
-	createSonarQubeFile(repositoryPayload)
-	//	githubRepository.Components.Sonarqube = *sonarQubeComponent
+	if os.Getenv("enable-sonarqube") == "true" { // we could check the token but i think thats the sonarqube libraries job
+		sonarQubeComponent, err2 := sonarqube.OnboardSonarQube(repositoryPayload)
```

Comment:
`err2` ?!?

Reply:
Fair enough. Add a linter to enforce no shadowing of errors? And maybe not name them `err`, `err2`, ... 
Maybe just my ocd talking, but that is ugly as shit imo

Reply:
whatr err inner or what? error errrroror haha